### PR TITLE
Fix/broken deactivation text

### PR DESCRIPTION
**Pull request by EstebanDalelR**
## Description
Users were seeing a deactivated notice when we found no error
## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)  
- [ ] New feature (non-breaking change which adds functionality)  
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)  
- [ ] Documentation update  
- [ ] Chore: cleanup/renaming, etc  
- [ ] RFC  
## Acceptance

- [x] I have read [the Contributing guidelines](CONTRIBUTING.md)
- [x] I have read the [Code of Conduct](CODE_OF_CONDUCT.md)

### CODE REVIEW

File: utils/actions/markdownHelpers/helper.ts [hunk 1]
```diff
@@ -21,12 +22,12 @@ const generalMarkdownHelper = ({
     markdown = `\n #### ${systemResponseName}`;
     markdown += (value?.data || [])
       .map(
-        ({ number, title, link, body }) =>
-          `\n - [#${number} - ${title}](${link}) \n`
+        ({ number, title, link, body, author, created_at }) =>
```
Comment (baristaGeek) [inline utils/actions/markdownHelpers/helper.ts:25]: Is this related?

**Review by baristaGeek — APPROVED** [review]

File: utils/actions/markdownHelpers/helper.ts (hunk 1 above)
Comment (EstebanDalelR) [reply utils/actions/markdownHelpers/helper.ts:25]: No, comes from #301 as to not have conflicts, when that one is merged there will be no diff
